iron-menu-behavior.html

```diff
@@ -90,16 +90,25 @@
     },
 
     /**
-     * Resets all tabindex attributes to the appropriate value based on the
+     * Reset attributes of the menu's new/changed collection of items:
+     *
+     * Reset all tabindex attributes to the appropriate value based on the
      * current selection state. The appropriate value is `0` (focusable) for
      * the default selected item, and `-1` (not keyboard focusable) for all
      * other items.
+     *
+     * Reset the aria-selected attribute based on the current selection state.
+     * As _applySelection notes, we need to set aria-selected to false for
+     * unselected items, even in their initial state, not just when they become
+     * deselected.
      */
-    _resetTabindices: function() {
+    _resetItemAttributes: function() {
       var selectedItem = this.multi ? (this.selectedItems && this.selectedItems[0]) : this.selectedItem;
 
       this.items.forEach(function(item) {
-        item.setAttribute('tabindex', item === selectedItem ? '0' : '-1');
+        var isSelected = item === selectedItem;
+        item.setAttribute('tabindex', isSelected ? '0' : '-1');
+        item.setAttribute('aria-selected', isSelected);
```

Comment:
Tab sets usually only have a small number of tabs. In the typical case of 3–7 tabs per tab set, I wouldn't expect an additional `setAttribute` per tab to have a noticeable impact on perf. Some tab-heavy designs (multiple tabbed regions on a single page, or tabs with tabs) might have more tabs — tens of tabs per page, maybe — but I'd still imagine that other costs would dominate.

It's possible that some attribute setting could be done lazily. E.g., with both attributes here, it **might** be possible to only set those attributes for the element with the focus and the elements on either side of it. But I'd imagine that could easily miss cases, and it seems safer to set the attributes when attached.